hasher-matcher-actioner/src/OpenMediaMatch/storage/interface.py

```diff
@@ -409,8 +410,19 @@ def bank_delete(self, name: str) -> None:
 
     # Bank content
     @abc.abstractmethod
-    def bank_content_get(self, id: t.Iterable[int]) -> t.Sequence[BankContentConfig]:
-        """Get the content config for a bank"""
+    def bank_content_get(
+        self, id: t.Iterable[int], signal_type: t.Optional[str] = None
```

Comment:
I'm trying to convince myself between adding this as the query here vs having a separate interface which takes the bank_content_id and only returns the signal types. 

Normally, we don't expect the users to be getting the signals in the same pass.

The key question is whether we think there is a usecase for fetching the signals without content_config, and I think there might be, but maybe not one worth optimizing for.

Reply:
Do you have an updated opinion on this? Tbh, w/o feedback from users, I'll just ditch it for now.

Reply:
Let's instead pass a bool to include all the signals
```suggestion
        self, id: t.Iterable[int], *, include_signals: bool = false
```